packages/connectivity/src/scp-cf/cache.ts

```diff
@@ -104,22 +94,12 @@ function isExpired<T>(item: CacheEntry<T>): boolean {
 }
 
 function inferExpirationTime(
-  expirationTime: DateInputObject | undefined
+  validityTimeInMs: number | undefined
 ): number | undefined {
-  return expirationTime
-    ? inferExpirationTimeFromDate(expirationTime)
+  const now = new Date();
+  return validityTimeInMs
+    ? new Date()
```

Comment:
```suggestion
    ? now
```
[req] This also applies here

Reply:
thank you for your review again. I forgot to replace it. I did it.